src/components/Footer.tsx

```diff
@@ -0,0 +1,61 @@
+import {
+  BottomNavigation,
+  Box,
+  Divider,
+  Link,
+  LinkProps,
+  Paper,
+  Typography,
+} from "@mui/material";
+import facebookLogo from "../assets/facebook.svg";
+import instagramLogo from "../assets/instagram.svg";
+
+export const Footer = () => {
```

Comment:
I like to add `React.FC` type for functional components objects:

```suggestion
export const Footer: React.FC = () => {
```